plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/navigator/database/StatisticsNavigatorNodeRenderer.java

```diff
@@ -192,6 +200,17 @@ private void drawObjectDescription(@NotNull GC gc, @NotNull DBNDatabaseNode node
         }
     }
 
+    private void drawObjectChildrenCounter(@NotNull GC gc, @NotNull DBNDatabaseNode node, @NotNull Rectangle bounds) {
+        int childCount = 0;
+        try {
+            childCount = node.getChildren(new VoidProgressMonitor()).length;
```

Comment:
```suggestion
            childCount = node.getChildren(new LocalCacheProgressMonitor(new VoidProgressMonitor())).length;
```
So it's guaranteed that no expensive reads will be performed